<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mtbTracker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../build-mtbTracker-Desktop-Debug/icon.ico</normaloff>../build-mtbTracker-Desktop-Debug/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="text">
             <string>Speichern</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="NewTourButton">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Neue Tour</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="twTouren">
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Datum</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Distanz</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Zeit</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Schnitt</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Höhenmeter</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Bemerkungen</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Gesamtstrecke:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leTotalDistance">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>65</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>1000</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Ziel:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leTargetDistance">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>65</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Verbleibend:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leRemainingDistance">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>65</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QProgressBar" name="pbRemaining">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionLog_Datei_ffnen"/>
    <addaction name="actionLog_Datei_speichern"/>
    <addaction name="actionLog_Datei_speichern_unter"/>
    <addaction name="separator"/>
    <addaction name="actionEinstellungen"/>
    <addaction name="separator"/>
    <addaction name="actionBeenden"/>
   </widget>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLog_Datei_ffnen">
   <property name="text">
    <string>Log-Datei öffnen</string>
   </property>
  </action>
  <action name="actionLog_Datei_speichern">
   <property name="text">
    <string>Log-Datei speichern</string>
   </property>
  </action>
  <action name="actionEinstellungen">
   <property name="text">
    <string>Einstellungen</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionLog_Datei_speichern_unter">
   <property name="text">
    <string>Log-Datei speichern unter ...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
